### DISCUSSION TITLE
Get ready for Game Off!

### DISCUSSION THREAD

**Discussion by leereilly** [community/community#37404] [Announcements]
<img src="https://user-images.githubusercontent.com/121322/198155790-8bd9bf35-ec78-4615-8cf6-49b409180b2c.png" alt="Game Off logo">

[Game Off](https://itch.io/jam/game-off-2022), our annual month-long game jam (or “hackathon for building games”) returns on November 1 for it’s 10th year! 🎉

It’s the perfect excuse to learn a new language, try out a new game engine, or even build your first game. Good luck, have fun, and [sign up now](https://itch.io/jam/game-off-2022).

**Comment by paltrowstar50** [comment]
Yeahyeahyeah

**Comment by Albert0n** [comment]
Well, I can say that I wasn't that impressed. In general, it's pretty hard to get entertained for me with games, for some reason.

**Comment by mehulrajdev** [comment]
impressive 
very good 
mark this as an answer

**Comment by reeveng** [comment]
Maybe we should write a game about designing good start screens? You could learn something from that then! https://github.com/orgs/community/discussions/categories/feed

**Comment by villinsian** [comment]
Getting ready for Game Off, a well-known game jam? You'll want to sharpen your coding and design skills, and perhaps look into interactive game mechanisms for inspiration. For a unique way to observe interactivity in real-time and get your creative juices flowing, check out the live gambling games at Tower.bet, found by typing [https://tower.bet/en-za/hi-lo](https://tower.bet/en-za/hi-lo) into the address bar. Their gaming platform could provide invaluable insights into user engagement and interactivity.

**Comment by novium2** [comment]
Retiring this post as it is over a year old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
